.github/workflows/get-environment.yml

```
@@ -0,0 +1,321 @@
+        id: latest_major_version
+        run: |
+          . centreon-develop/.version
+          echo "latest_major_version=$MAJOR" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - if: ${{ github.event_name == 'pull_request' }}
+        name: Get nested pull request path
+        id: pr_path
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
```

actionlint

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)